src/enc/pm_tables.c

```diff
@@ -336,6 +336,30 @@ static uint8_t pm_encoding_ibm862_table[256] = {
     0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, // Fx
 };
 
+/**
+ * Each element of the following table contains a bitfield that indicates a
+ * piece of information about the corresponding IBM864 character.
+ */
+static uint8_t pm_encoding_ibm864_table[256] = {
+//  0  1  2  3  4  5  6  7  8  9  A  B  C  D  E  F
+    0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, // 0x
```

Comment:
These values are the same as ibm862 but I think that's right based on what I see on the wikipedia page. https://en.wikipedia.org/wiki/Code_page_864